src/libgraphviz/lib.rs

```diff
@@ -533,10 +534,10 @@ impl<'a> LabelText<'a> {
     /// Renders text as string suitable for a label in a .dot file.
     /// This includes quotes or suitable delimiters.
     pub fn to_dot_string(&self) -> String {
-        match self {
-            &LabelStr(ref s) => format!("\"{}\"", s.escape_default()),
-            &EscStr(ref s) => format!("\"{}\"", LabelText::escape_str(&s)),
-            &HtmlStr(ref s) => format!("<{}>", s),
+        match *self {
```

Comment:
I believe match\_default\_bindings has eliminated the need of `*` like this.

Reply:
True, it would compile without it too; while it's certainly easier to let the compiler do the dereferencing on its own, I wonder if doing it manually isn't faster :thinking:.